doc/radosgw/metrics.rst

```diff
@@ -117,11 +98,51 @@ For a user ``User1`` doing operations on ``Bucket1``, the RGW Op Metrics can be
         ...
     ]
 
+Enabling the Radosgw User & Bucket Counter Caches
+=================================================
+
+To track op metrics by user the Radosgw the config value ``rgw_user_counters_cache`` must be set to ``true``. 
+
+To track op metrics by bucket the Radosgw the config value ``rgw_bucket_counters_cache`` must be set to ``true``. 
```

Comment:
this section would be a good place to discuss the scaling concerns for systems with many users/buckets, and why these features default to off